README.md

```diff
@@ -144,10 +144,11 @@ To build the Open XML SDK
 
 If you want to use a command line approach:
 
-1. Go to the directory that contains the solution.
+1. Go to the directory that contains the project file, i.e. `cd <clone path>\src\DocumentFormat.OpenXml`
 1. Run `dotnet restore` in the directory.
-1. Run `dotnet test DocumentFormat.OpenXml.Tests` to run the tests.
-1. Run `dotnet pack DocumentFormat.OpenXml` to generate a nupkg.
+1. Run `dotnet test DocumentFormat.OpenXml.Tests.csproj` to run the tests.
```

Comment:
This should be changed to run it from the solution directory with `dotnet test` ... it will run all the tests. This will only run a single one.